docs/ref/class-based-views/generic-editing.txt

```diff
@@ -275,12 +275,26 @@ editing content:
     * :class:`django.views.generic.base.TemplateResponseMixin`
     * :class:`django.views.generic.edit.BaseDeleteView`
     * :class:`django.views.generic.edit.DeletionMixin`
+    * :class:`django.views.generic.edit.FormMixin`
+    * :class:`django.views.generic.base.ContextMixin`
     * :class:`django.views.generic.detail.BaseDetailView`
     * :class:`django.views.generic.detail.SingleObjectMixin`
     * :class:`django.views.generic.base.View`
 
     **Attributes**
 
+    .. attribute:: form_class
+
+        Inherited from :class:`~django.views.generic.edit.BaseDeleteView`. The
```

Comment:
We could drop this sentence, but I think this is the right place to document the attribute (rather than on `BaseDeleteView`)